lib/improver/nbhood/recursive_filter.py

```diff
@@ -337,13 +339,13 @@ def process(self, cube, alphas_x=None, alphas_y=None, mask_cube=None):
                     output, mask_cube))
             mask = mask.data.squeeze()
 
-            padded_cube = SquareNeighbourhood().pad_cube_with_halo(
-                output, self.edge_width, self.edge_width)
+            padded_cube = pad_cube_with_halo(
+                output, 2*self.edge_width, 2*self.edge_width)
```

Comment:
It would be good to update the docstring for `edge_width` within the `__init__`, as it refers to the "SquareNeighbourhood halo", but perhaps this should just be "halo". What do you think?

Reply:
Done